test/modules/encoders/test_clip_text_encoder.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from test.test_utils import assert_expected, set_rng_seed
+from torchmultimodal.modules.encoders.clip_text_encoder import CLIPTextEncoder
+
+
+class TestCLIPTextEncoder:
+    @pytest.fixture
+    def setUp(self):
+        set_rng_seed(1234)
+        context_length = 77
+        return context_length
```

Comment:
How about using `pytest.fixture` to initialize `CLIPTextEncoder` as it's needed by every test? Although you'll need to find a way to pipe in runtime arguments such as `use_clip_init`. Return a function is just one way to do so:

https://github.com/facebookresearch/multimodal/blob/5cbf74edfba952005439cef3ac25eb10882c9302/test/modules/layers/test_attention.py#L13-L22